css/style.css

```diff
@@ -1,18 +1,15 @@
 body {
     background-color: #2e2e2e;
-    color: #CCC;
-    font-family: 'Droid Sans', "Lucida Grande", "Lucida Sans", "Liberation Sans", sans-serif;
-    font-size: 0.8em;
+    color: #ddd;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";;
```

Comment:
I’d prefer to keep using Droid Sans. You describe the change as using fonts that are more familiar to the user, but the opposite is the case: on Linux (particularly in i3), many users won’t have the concept of an operating system font, so no choice will be particularly familiar. Can we keep Droid Sans and Mako as the first choice please? The rest is fine to adjust.

Reply:
You're right. I hadn't considered the fact that the i3 userbase is mostly Linux users without a unified system font. Added a revert commit.